celesta.js

```diff
@@ -1296,10 +1298,16 @@
                 }
                 break;
             case KEYCODE_SPACE:
-                if (this._config.space_key_open) {
-                    this.open();
-                    // Don't prevent space keypress when already open (used for typed search or default browser action)
-                    if (!this._is_open) {
+                if (this._is_open) {
+                //if we not in the middle of typing, select hovered option
```

Comment:
comment codestyle notices:
* add space before first letter (`// if` not `//if`)
* begin full-phrase comments with capital letter (`If` not `if`)